journeyStar_shacl.ttl

```diff
@@ -148,6 +148,36 @@ js-shacl:JourneyShape a sh:NodeShape ;
         sh:node js-shacl:LocationShape ;
     ] .
 
+### Shape for Class js:Journey
```

Comment:
I have created a shape for the superclass without cardinalities. This way we should only need to define the cardinalities here.